draft-todo-yourname-protocol.md

```diff
@@ -24,6 +24,13 @@ abbrev: "TODO - Abbreviation"
 category: info
 
 docname: draft-todo-yourname-protocol-latest
+submissiontype: IETF
+stand_alone: yes
+ipr: trust200902
+pi: [toc, sortrefs, symrefs]
```

Comment:
These are covered by `v: 3`, so I don't want to repeat them.  See https://mailarchive.ietf.org/arch/msg/rfc-markdown/JC__LDDGuUbSFqyaEntF9r4kwKw/

The alternative values for these are specialist tools, so even including a stub doesn't work.

Reply:
Thanks, I didn't know that about `v: 3`. I've removed these lines

Reply:
V : 3 🚫❗